frontend/util/city-input.component.tsx

```diff
@@ -56,6 +57,7 @@ export default function CityInput(props) {
   );
 
   function setCityAndTab(city) {
+    console.log(city);
```

Comment:
We should not commit or push console.log()'s. I'm actually surprised that this passed the eslint check, I thought we had a rule in there that disallows console.log in committed code.